Channels.hs

```diff
@@ -23,7 +23,9 @@ import qualified Data.Text as DT
 import qualified Data.ByteString.Char8 as C8
 import Data.Text (pack, unpack, replace, Text)
 import qualified Network.Wreq as W
-import Network.HTTP.Client (HttpException(StatusCodeException))
+import Network.Wreq.Session (Session)
+import qualified Network.Wreq.Session as WS
+import Network.HTTP.Client -- (HttpException(StatusCodeException))
```

Comment:
Why was this commented out? :) Let's namespace it or remove the import all together.

Reply:
Sorry, that sneaked in by accident when I inadvertently set up an environment with http-client >= 0.5.0 where the type of `HttpException` has changed, rather than using stack.yaml provided.